src/Studio/Plugins/Devices/BciDeviceWidget.cpp

```diff
@@ -135,12 +129,10 @@ void BciDeviceWidget::AddSensorItems(QTreeWidgetItem* parent)
 		// add signal quality subitem
 		item = new QTreeWidgetItem(sensorItem);
 		item->setText(0, "Signal Quality");
-		item->setHidden(NEUROMORE_MINIMAL_DEVICE_STATS);
-
+		
```

Comment:
Please don't touch unrelated PR code.

Reply:
@dioptre please checkout to the latest commit I have updated my code with the latest version.